tests/fixtures/issues/issue-1253.php

```diff
@@ -6,7 +6,7 @@
         'properties' => [
             'Comment' => 'Kindle',
             'Browser' => 'Kindle',
-            'Browser_Type' => 'unknown',
+            'Browser_Type' => 'Browser',
```

Comment:
Figured the Kindle UAs should probably be a Browser, so updated that and had to modify these tests to match.